pom.xml

```diff
@@ -293,8 +294,8 @@
         <artifactId>maven-compiler-plugin</artifactId>
         <version>3.5.1</version>
         <configuration>
-          <source>1.8</source>
-          <target>1.8</target>
+          <source>11</source>
```

Comment:
as of we are mentioning that most recent version requires Java 11, to be able to write code in Java 11 I decided to update it also